src/DETHRACE/common/sound.c

```diff
@@ -16,13 +16,13 @@
 #include "s3/s3_brender.h"
 #include "utility.h"
 
-int gSound_detail_level;
+int gSound_detail_level = 1;
 int gVirgin_pass = 1;
 int gOld_sound_detail_level = -1;
-int gLast_tune;
-int gRandom_MIDI_tunes[3];
-int gRandom_Rockin_MIDI_tunes[3];
-int gRandom_CDA_tunes[7];
+int gLast_tune = -1;
+int gRandom_MIDI_tunes[3] = { 9500, 9501, 9502 };
+int gRandom_Rockin_MIDI_tunes[3] = { 9500, 9501, 9502 };
+int gRandom_CDA_tunes[7] = { 9600, 9601, 9602, 9603, 9604, 9605, 9606 };
```

Comment:
See https://github.com/dethrace-labs/dethrace/pull/393/files#r1676807636
A similar change is needed in `sound.h`.

```suggestion
int gRandom_CDA_tunes[8] = { 9600, 9601, 9602, 9603, 9604, 9605, 9606, 9607 }; /* dethrace: Changed to size 8 */
```